packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -476,6 +501,83 @@ export default decorate([
               </div>
             </fieldset>
           )}
+          {(type === 'azure' || type === 'azurite') && (
+            <fieldset className='form-group form-group'>
+              {state.type === 'azurite' && (
+                <div className='input-group form-group'>
+                  <span className='align-middle '>
+                    {_('remoteS3LabelAllowInsecure')}{' '}
+                    <Tooltip content={_('remoteS3TooltipAcceptInsecure')}>
+                      <Icon icon='info' size='lg' />
+                    </Tooltip>
+                  </span>
+                  <Toggle
+                    className='align-middle pull-right'
+                    disabled={protocol !== 'https'}
+                    onChange={effects.setAllowUnauthorized}
+                    value={allowUnauthorized}
+                  />
+                </div>
+              )}
+              <div className='input-group form-group'>
+                <input
+                  className='form-control'
+                  name='host'
+                  onChange={effects.linkState}
+                  placeholder={`Host (e.g. ${state.type === 'azurite' ? '<username>.blob.core.windows.net' : '127.0.0.1:10000'})`}
```

Comment:
```suggestion
                  placeholder={`Host (e.g. ${state.type === 'azurite' ? '127.0.0.1:10000' : '<username>.blob.core.windows.net'})`}
```

Reply:
I think it is the opposite : Azurite is for testing purpose, usually in localhost whereas Azure is, for now, hosted in windows.net